frontend/src/app/containers/CreateSongPage/index.tsx

```diff
@@ -1,7 +1,149 @@
+
+  return (
+    <div data-testid="CreateSongPage" className="flex flex-col items-center">
+      <h3 className="text-lg font-medium leading-6 text-gray-900">
+        Upload Song
+      </h3>
+      <p className="mt-1 text-sm text-gray-600">
+        Upload a song link to create covers!
+      </p>
+
+      <div className="w-full sm:w-3/4 m-5">
+        <form
+          onSubmit={onSubmitForm}
+          className="shadow border overflow-hidden sm:rounded-md"
+        >
+          <div className="px-4 py-5 bg-white sm:p-6">
+            <div className="grid grid-cols-6 gap-6">
+              <div className="col-span-6">
+                <label htmlFor="title" className={styles.label}>
+                  Title
+                </label>
+                <input
+                  type="text"
+                  name="title"
+                  id="title"
+                  value={Form.title}
+                  onChange={e => onChangeForm(e, 'title')}
+                  className={styles.input}
+                />
+              </div>
+
+              <div className="col-span-6">
+                <label htmlFor="artist" className={styles.label}>
+                  Artist
+                </label>
+                <input
+                  type="text"
+                  name="artist"
+                  id="artist"
+                  value={Form.artist}
+                  onChange={e => onChangeForm(e, 'artist')}
+                  className={styles.input}
+                />
+              </div>
+
+              <div className="col-span-6 sm:col-span-3">
+                <label htmlFor="category" className={styles.label}>
+                  Category
+                </label>
+                <select
+                  name="category"
+                  id="category"
+                  value={Form.category}
+                  onChange={e => onChangeForm(e, 'category')}
+                  className="mt-1 block w-full py-1 border-gray-300 bg-white rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
+                >
+                  {categories.map((item, index) => (
+                    <option data-testid="option" value={index} key={index}>
+                      {item}
+                    </option>
+                  ))}
+                </select>
+              </div>
+
+              <div className="col-span-6">
+                <label htmlFor="reference" className={styles.label}>
+                  Reference Link
+                </label>
+                <input
+                  type="text"
+                  name="reference"
+                  id="reference"
+                  value={Form.reference}
+                  onChange={e => onChangeForm(e, 'reference')}
+                  className={styles.input}
+                />
+              </div>
+
+              <div className="col-span-6">
+                <label htmlFor="description" className={styles.label}>
+                  Description
```

Comment:
```suggestion
                  Description(optional)
```
Description은 안써도 넘어갈 수 있는데 선택사항인걸 표시해 주는건 어떨까요?

Reply:
수정했습니다!